src/schema/types/application.type.ts

```diff
@@ -244,6 +246,65 @@ export const ApplicationType = new GraphQLObjectType({
     distributionLists: {
       type: new GraphQLList(DistributionListType),
     },
+    autoAssignedUsers: {
```

Comment:
@ketanreliefapps one thing to change there:
- use UserConnection as we do on the role.type
- edit the users field of the application type so it also use the UserConnection instead of GraphQLList
- remove the usersCount field